sign-in.txt

```diff
@@ -2,5 +2,5 @@
 //TODO: Be sure you have created a feature branch from develop 
 //TODO: Please put your name and github username AS SHOWN below ... 
 //       ... then commit and create pull request to develop
-
+Daniel Watkins : Daniel-Watkins
```

Comment:
Daniel -shouldn't this come after my name

Reply:
Hello